modules/sftp/remote-state.tf

```diff
@@ -0,0 +1,9 @@
+module "vpc" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "0.22.4"
+
+  component = "vpc"
+  stage     = "${var.tenant}-${var.stage}"
```

Comment:
this looks like its old, shouldn't stage just be stage, and tenant the tenant. which is all set from context.

```suggestion
```